src/sql/src/session/vars/definitions.rs

```diff
@@ -1356,6 +1356,27 @@ pub static ARRANGEMENT_EXERT_PROPORTIONALITY: VarDefinition = VarDefinition::new
     false,
 );
 
+pub static ENABLE_TIMELY_ZERO_COPY: VarDefinition = VarDefinition::new(
+    "enable_timely_zero_copy",
+    value!(bool; false),
+    "Enable the zero copy allocator (timely dataflow).",
+    false,
+);
+
+pub static ENABLE_TIMELY_ZERO_COPY_LGALLOC: VarDefinition = VarDefinition::new(
+    "enable_timely_zero_copy_lgalloc",
+    value!(bool; false),
+    "Enable backing the zero copy allocator with lgalloc (timely dataflow).",
+    false,
+);
+
+pub static TIMELY_ZERO_COPY_LIMIT: VarDefinition = VarDefinition::new(
+    "timely_zero_copy_limit",
+    value!(Option<usize>; None),
```

Comment:
Open Q but should this default to `None` or perhaps `Some(0)` or something "conservative"? It's probably fine, because we don't opt in to zero copy.

Reply:
Yup, we should keep this in mind for when we might enable zero-copy.